### PR TITLE
Disabled waiting for first data on iOS by default and timeout parameter added

### PR DESCRIPTION
**Pull request by srmncnk**
This PR addresses the issue with latest versions not working on iOS: https://github.com/ysak-y/flutter_audio_capture/issues/16

By default, iOS no longer waits for first data to arrive. Instead a parameter is added for this behaviour. A timeout parameter is also added in case we have to wait longer on iOS for the data to arrive.

### CODE REVIEW

**Comment by srmncnk** [comment]
This should be merged only after @vladsonkin has confirmed it works for him.

**Comment by kijka** [comment]
@srmanc it works! Thanks for the Christmas present :]

If the "waiting for the first data" feature doesn't work reliably on iOS, then maybe we can fully delete it? Otherwise, functionality could differ per platform. Or maybe add a comment that this flag on iOS can be buggy.

@ysak-y could you create a new release, please?

**Review by kijka — APPROVED** [review]

**Comment by srmncnk** [comment]
Yeah, sure, comment added. https://github.com/ysak-y/flutter_audio_capture/pull/19/commits/7bea0c86e689cc74a115a3b1bf73d47ef3f84ee1

**Review by ysak-y — APPROVED** [review]
Nice, thanks! Please merge it!

**Comment by srmncnk** [comment]
@ysak-y If you want me to do it, i need write access to this repository. Happy to do it though.

**Comment by ysak-y** [comment]
@srmanc Hi, I've invited you as the collaborator. Please approve if you want, you'll be permitted to merge it later.

**Pull request merged**
